graphql_compiler/schema_transformation/split_query.py

```diff
@@ -564,41 +739,45 @@ def _get_child_query_node_and_out_name(ast, child_type_name, child_field_name, n
     )
     # Get new child_selections by replacing or adding in new property field
     child_property_fields_map, child_vertex_fields = _split_selections_property_and_vertex(
-        child_selections
+        list(child_selections)
```

Comment:
Assuming this is the same kind of issue? Can we solve it by typing the callee with `Sequence` instead?